draft-ietf-ipsecme-ikev2-sa-ts-payloads-opt.md

```diff
@@ -194,16 +194,10 @@ HDR, SK {N(REKEY_SA,oldSPI), N(OPTIMIZED_REKEY,newSPIi),
                                          Nr, [KEr,]}
 ~~~~
 
-For the initial Child SA that was negotiated as part of an initial IKE exchange (e.g., IKE_AUTH), at the time of its creation the parameters of PFS and KE method for Child SAs are not negotiated. Therefore, the KE method for the initial IKE SA should also be recognized as the one for this initial Child SA.
+For the initial Child SA that was negotiated as part of an initial IKE exchange (e.g., IKE_AUTH), at the time of its creation the parameters of PFS and KE method for Child SAs are not negotiated.
+Therefore, when rekeying the initial Child SA for the first time, a regular rekey MUST be used for the negotiation of these parameters. Then, the next rekey can use the optimized way.

```

Comment:
i don't want to unconditionally say that, as i am still hoping to kill the regular rekey.